libraries/TGRSIAnalysis/TTip/TTipHit.cxx

```diff
@@ -38,8 +41,10 @@ bool TTipHit::InFilter(Int_t wantedfilter) {
 }
 
 void TTipHit::Clear(Option_t *opt) {
-   filter = 0;
-   fPID   = 0;
+   filter 		= 0;
+   fPID   		= 0;
+   tip_channel 	= 0;
+   time_fit		= 0;
```

Comment:
What's a tip_channel? Can you get away with using the fdetector of TGRSIDetectorHIt? Or is it something separate? Also, do you wish to store the waveform fit variables to disk to playback later? I only ask because the less variables you write to disk the faster it will run. I don't deal with Tigress data, so I generally don't care, just trying to help out where I can.